.github/scripts/build-test.sh

```diff
@@ -1,18 +1,35 @@
 #! /usr/bin/env bash
 #
-# Wrapper script to build prmon on travis and run all tests
+# Wrapper script to build prmon and run all tests
 #
 # Note that CXX, CC and CMAKE can all be customised so that
 # they can be matched to the development environment in each
 # container; CMAKE_EXTRA can be used to pass special options
 # to cmake for particular platforms
 cd /tmp
+echo "Starting build and test for platform $PLATFORM, compiler suite $COMPILER"
 if [ -z "$CXX" ]; then
-    CXX=$(type -p g++)
+	if [ "$COMPILER" == "clang" ]; then
+		CXX=$(type -p clang++)
+	else
+		# gcc suite is the fall through
+    	CXX=$(type -p g++)
```

Comment:
Admittedly this is my OCD kicking in but can we indent this properly? 😄

Reply:
It seemed to be ok for me, but it was tabs - changed to space indentation (4 spaces), so then that should be more consistent